arthur_bench/scoring/summary_quality.py

```diff
@@ -25,7 +26,13 @@
 logger = logging.getLogger(__name__)
 
 
-def truncate_input_text(input_text, ref_output, cand_output) -> Tuple[str, bool]:
+def truncate_input_text(
+    input_text,
+    ref_output,
+    cand_output,
+    context_window: int = CONTEXT_WINDOW_MAP[EVALUATOR_MODEL],
+    tiktoken_encoder: Encoding = TIKTOKEN_ENCODER,
```

Comment:
a general naming / typing question throughout - the `tiktoken` library is optimized for openai models so I think it would be clearer to rename this to `tokenizer` or similar for future configurability

Reply:
this function should be able to accept any tokenizer that has an encode method

Reply:
Renamed to "tokenizer", but I'm not sure what you mean on the second comment? Is there a different type I should be using? Or `Encoding` is fine in the dynamically typed world of python? I'm not sure if there is a type I could be using instead that is just defining the methods that need to be on the object.